src/cephadm/cephadm

```diff
@@ -2042,22 +2063,7 @@ def command_bootstrap():
     if not args.skip_dashboard:
         logger.info('Enabling the dashboard module...')
         cli(['mgr', 'module', 'enable', 'dashboard'])
-
-        # wait for the service to become available
-        logger.info('Waiting for the dashboard to start...')
-        def is_dashboard_available():
-            # type: () -> bool
-            timeout=args.timeout if args.timeout else 30 # seconds
```

Comment:
intentionally dropping a timeout for the cli commands?

Reply:
yeah, these calls aren't really any different than the other cli commands, which don't have timeouts.